ion_phys/tests/test_stim.py

```diff
@@ -0,0 +1,25 @@
+from ion_phys.ions.ca43 import Ca43, ground_level, P12, D32
+from ion_phys import Laser
+from ion_phys.rate_equations import Rates
+import unittest
+
+class TestGamma(unittest.TestCase):
+    def test_multi_transition(self):
+        """Test for no errors with lasers on multiple transitions"""
```

Comment:
"Test for no errors" is an odd description. Maybe include a URL for the relevant issue?